src/integrationtest/python/DocTraversals_tests.py

```diff
@@ -0,0 +1,68 @@
+# Name: Debasish Kanhar
```

Comment:
These integration test files should belong to the other pull request. I might be easier to try and merge PR #33 to initial-pr and then do a rebase to clean this up.